examples/add_public_raster_dataset.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Add Public Raster Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geoengine as ge\n",
+    "from datetime import datetime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize Geo Engine as Admin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ge.initialize(\"http://localhost:3030\", admin_token=\"8aca8875-425a-4ef1-8ee6-cdfc62dd7525\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## List volumes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[Volume(name='test_data', path='test_data/')]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "volumes = ge.volumes()\n",
+    "volumes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define metadata"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'type': 'GdalStatic',\n",
+       " 'time': None,\n",
+       " 'params': {'filePath': 'raster/landcover.tif',\n",
+       "  'rasterbandChannel': 1,\n",
+       "  'geoTransform': {'originCoordinate': [-180.0, 90.0],\n",
+       "   'xPixelSize': 0.1,\n",
+       "   'yPixelSize': -0.1},\n",
+       "  'width': 3600,\n",
+       "  'height': 1800,\n",
+       "  'fileNotFoundHandling': <FileNotFoundHandling.NODATA: 'NoData'>,\n",
+       "  'noDataValue': None,\n",
+       "  'propertiesMapping': None,\n",
+       "  'gdalOpenOptions': None,\n",
+       "  'gdalConfigOptions': None,\n",
+       "  'allowAlphabandAsMask': True},\n",
+       " 'resultDescriptor': Data type:         U8\n",
+       " Spatial Reference: EPSG:4326\n",
+       " Measurement:       Land Cover (0: Water Bodies, 1: Evergreen Needleleaf Forests, 2: Evergreen Broadleaf Forests, 3: Deciduous Needleleaf Forests, 4: Deciduous Broadleleaf Forests, 5: Mixed Forests, 6: Closed Shrublands, 7: Open Shrublands, 8: Woody Savannas, 9: Savannas, 10: Grasslands, 11: Permanent Wtlands, 12: Croplands, 13: Urban and Built-Up, 14: Cropland-Natural Vegetation Mosaics, 15: Snow and Ice, 16: Barren or Sparsely Vegetated)}"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "file_path = \"raster/landcover.tif\" # specify file path relative to volume path\n",
```

Comment:
Please check that this really works.

![image](https://user-images.githubusercontent.com/4947061/208492278-0ff15895-37ce-4578-ac4a-9f6f4508e64a.png)

Maybe there is just another path? We should have one example that works out of the box.

Reply:
0c333e698ae5a950437050af9f66aad1f32fdfc3

Reply:
It occurs now in the notebook.
But when I go to the UI, the same error occurs.

![image](https://user-images.githubusercontent.com/4947061/208607396-fc1314e3-8f1d-47bb-8b2d-ebbb9364f8e8.png)